cves/CVE-2007-0404.yml

```diff
@@ -126,8 +141,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: The reason this vulnerability occured was because the code writer did not test or check or
```

Comment:
I would change "code writer" to "developer"

Reply:
I would try to reduce the redundancy of "did not test or check or take security seriously enough"